Moderation-Policy.md

```diff
@@ -20,29 +20,11 @@ moderation request, please see [Requesting Moderation][]
 
 ## Applicability
 
-By default, this policy applies to all repositories under the Node.js GitHub
+This policy applies to all repositories under the Node.js GitHub
 Organization and all Node.js Working Groups. This policy also applies to the
 [Node.js Slack Community](https://node-js.slack.com), supported by the Admin
 team of the Slack organization.
```

Comment:
Maybe a separate issue, but does the OpenJS Foundation Slack have its own moderation policy, or would certain channels pertaining to Node.js (e.g. #nodejs-*) that are on it also fall under this moderation policy?

Reply:
I think @brianwarner, @jorydotcom, or @rginn would have to indicate what (if any) is the formalized moderation policy for the OpenJS Foundation Slack. It is unlikely to be our moderation policy because I don't think it's practical to have different moderation policies for different channels inside the OpenJS Foundation Slack, but that's really up to them, I think.